composables/auto-prompter.ts

```diff
@@ -237,9 +238,18 @@ export function useAutoPrompter() {
         const a = randomlySelectFromDistribution()
         const b = randomlySelectFromDistribution(a)
 
+        const adversaryA = activeCandidates.find(candidate => candidate.id === Number(a))
+        const adversaryB = activeCandidates.find(candidate => candidate.id === Number(b))
+
+        return [adversaryA, adversaryB]
+    }
+
+    async function  runBattle(adversaries: [Candidate, Candidate]) {
```

Comment:
```suggestion
    async function  runBattle(adversaryA: Candidate, adversaryB: Candidate) {
```
I would change it into a more standard way of passing down the parameters, instead of a tuple